webcompat/static/css/development/components/button.css

```diff
@@ -26,6 +26,7 @@
   background-image: none;
   user-select: none;
   -webkit-appearance: none;
+  margin: 0 1em .5em 1em;
```

Comment:
Maybe wou can add an media querie for this margin, to avoid oveflow on small screen.

![capture d ecran 2017-01-12 a 06 41 12](https://cloud.githubusercontent.com/assets/1997108/21878371/2156dd7a-d892-11e6-874d-77bf47febfd8.png)